addon/components/cfb-form-editor/question/default.js

```diff
@@ -0,0 +1,94 @@
+import { getOwner } from "@ember/application";
+import { computed } from "@ember/object";
+import RenderComponent from "ember-validated-form/components/validated-input/-themes/uikit/render";
+
+import layout from "../../../templates/components/cfb-form-editor/question/default";
+
+import { TYPE_MAP } from "ember-caluma/lib/field";
+
+export default RenderComponent.extend({
+  layout,
+
+  subComponent: computed("model.__typename", function () {
+    const type = this.model.__typename.replace("Question", "").toLowerCase();
+    return ["choice", "multiplechoice"].includes(type) ? "powerselect" : type;
+  }),
```

Comment:
Why aren't we using the `cf-field` components?

Reply:
I'll have to check this decision. This probably comes from the time before I mocked lib/field instead of just lib/answer and lib/question.

Reply:
Using `cf-field` is too specific.

![Screenshot_20201223_123038](https://user-images.githubusercontent.com/65539425/102994728-e7314280-451f-11eb-98f9-d1309d6d1554.png)

As a compromise I switched to `{{component (get-widget` which seems to work. Not sure, if I should set a Power Select override for (Multiple)Choice. I'll look into how I would do that.

![Screenshot_20201223_125110](https://user-images.githubusercontent.com/65539425/102994858-34adaf80-4520-11eb-9276-ad413b891a38.png)

![Screenshot_20201223_125202](https://user-images.githubusercontent.com/65539425/102994859-35dedc80-4520-11eb-8f17-82cc9e18d9b8.png)

Reply:
Ok. I found a simple way to set Power Select.